packages/shared/src/components/utilities.tsx

```diff
@@ -70,10 +71,22 @@ const themeColors = {
     button: 'btn-primary',
   },
 };
+
 export const getThemeColor = (color: string, fallback: string): ThemeColor => {
   return themeColors[color] ?? themeColors[fallback];
 };
 
+const themeFonts = {
+  bodyBold: 'font-bold typo-body',
+  body: 'typo-body',
+  title: 'typo-title3',
+  titleBold: 'font-bold typo-title3',
```

Comment:
I suggest the same here
```suggestion
  title: 'typo-title3',
  title3Bold: 'font-bold typo-title3',
```

Reply:
I changed the names 👍  @rebelchris Is it ok to use a number in a variable name or should we as an actual word for it?

Reply:
All good for me 🙌